src/main/java/uk/gov/companieshouse/insolvency/delta/validation/InsolvencyDeltaValidator.java

```diff
@@ -44,6 +44,9 @@ public void validateCaseDates(Insolvency insolvency) throws Exception {
     private boolean checkFieldAgainstSchema(Field[] fields, CaseNumber caseNumber,
                                             List<String> listOfFieldsToValidate) throws Exception {
         for (Field field: fields) {
+            if (field.isSynthetic()) {
```

Comment:
Might this be cleaner if it was `! field.isSynthetic()` and covered the whole code block

Reply:
It would be adding another level nesting in a method with a for loop and another if check that would be wrapped by the if check. Personally I think  a smaller if check makes the code more readable.